app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+ class UsersController < ApplicationController
+
+  def new
+    redirect_to home_path if current_user
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to login_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+    def user_params
```

Comment:
That would only be the case if you have `private` outdented to the same level as `class`/`end`, which would be fine. If you do not do that, then they should be at the same indentation as `private`. All of the method definitions should be the same indentation, both above and below the `private`.